kopi/assetlist.json

```diff
@@ -0,0 +1,497 @@
+      "name": "XKP",
+      "display": "XKP",
+      "symbol": "XKP",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/kopi.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/kopi.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm",
+        "staking",
+        "automations"
+      ],
+      "socials": {
+        "twitter": "https://x.com/kopi_money",
+        "discord": "https://discord.gg/kopi-money"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "A Stablecoin native to the Kopi blockchain",
+      "denom_units": [
+        {
+          "denom": "ukusd",
+          "exponent": 0
+        },
+        {
+          "denom": "kUSD",
+          "exponent": 6
+        }
+      ],
+      "base": "ukusd",
+      "name": "kUSD",
+      "display": "kUSD",
+      "symbol": "KUSD",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/kusd.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/kusd.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm",
+        "automations",
+        "arbitrage"
+      ],
+      "socials": {
+        "twitter": "https://x.com/kopi_money",
+        "discord": "https://discord.gg/kopi-money"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "A MM deposit receipt of the Stablecoin native to the Kopi blockchain",
+      "denom_units": [
+        {
+          "denom": "uckusd",
+          "exponent": 0
+        },
+        {
+          "denom": "ckUSD",
+          "exponent": 6
+        }
+      ],
+      "base": "uckusd",
+      "name": "ckUSD",
+      "display": "ckUSD",
+      "symbol": "CKUSD",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/kusd.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/kusd.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm"
+      ],
+      "socials": {
+        "twitter": "https://x.com/kopi_money",
+        "discord": "https://discord.gg/kopi-money"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "A MM deposit receipt of the Stablecoin USDC from noble Chain",
+      "denom_units": [
+        {
+          "denom": "ucusdc",
+          "exponent": 0
+        },
+        {
+          "denom": "cUSDC",
+          "exponent": 6
+        }
+      ],
+      "base": "ucusdc",
+      "name": "cUSDC",
+      "display": "cUSDC",
+      "symbol": "CUSDC",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/non-cosmos/ethereum/images/usdc.svg"
```

Comment:
The URLs referring to a non-cosmos asset are missing the underscore _ before the non-cosmos (should be '_non-cosmos')